### ISSUE TITLE
Add publisher information to footer

### ISSUE THREAD

**Issue by OssianEriksson** [ECarlsson/ftek-foton-gallery#21]
The "informationsansvarig" of the division should be mentioned as publisher somewhere on the page.

**Comment by ghost** [comment]
Agreed; and most likely the entire footer will need a remake when the ftek.se eventually is remade as to better fit the theme. Would ideally not be hardcoded but rather added with environment variables or similar so that it can be updated without pushing updates to the entire project.
